react/features/remote-video-menu/components/web/MuteButton.js

```diff
@@ -87,34 +56,12 @@ class MuteButton extends Component<Props> {
                 displayClass = { muteConfig.muteClassName }
                 iconClass = 'icon-mic-disabled'
                 id = { `mutelink_${participantID}` }
-                onClick = { this._onClick } />
+                // eslint-disable-next-line react/jsx-handler-names
```

Comment:
can't we avoid this by calling the handler `onSomethingClick` ?

Reply:
The name ```_handleClick``` comes from the ```AbstractButton``` implementation. So I either had to do this, or encapsulate (or re-bind) it into a meaningless function that starts with ```_on``` but simply invokes ```_handleClick```. Both are kind of pointless, but I picked this solution to avoid unnecessary code. I was thinking: the entire render method will be removed anyhow when we refactor web to use the same abstraction.

Reply:
Ok.